src/test/scala/mesosphere/marathon/api/akkahttp/v2/AppsControllerTest.scala

```diff
@@ -106,6 +106,18 @@ class AppsControllerTest extends UnitTest with GroupCreation with ScalatestRoute
       (body, plan)
     }
 
+    def addAppsToRootGroupRootGroup(apps: Seq[App]): (Seq[AppDefinition], DeploymentPlan, RootGroup) = {
+      val normed = apps.map(normalize)
+      val appDefs = normed.map(Raml.fromRaml(_))
```

Comment:
What's happening here? Why do we have to normalize apps and the transform them back from the RAML?